cli/args_test.go

```diff
@@ -373,18 +374,19 @@ func TestTerraformHelp(t *testing.T) {
 		args     []string
 		expected string
 	}{
-		{[]string{"terragrunt", "plan", "--help"}, "Usage: terraform plan"},
-		{[]string{"terragrunt", "apply", "-help"}, "Usage: terraform apply"},
-		{[]string{"terragrunt", "apply", "-h"}, "Usage: terraform apply"},
+		{[]string{"terragrunt", "plan", "--help"}, "Usage: terraform .* plan"},
+		{[]string{"terragrunt", "apply", "-help"}, "Usage: terraform .* apply"},
+		{[]string{"terragrunt", "apply", "-h"}, "Usage: terraform .* apply"},
```

Comment:
Terraform's help text is a bit different now, so had to update this test accordingly.